### DISCUSSION TITLE
Disable Dashbords ... Feature Request

### DISCUSSION THREAD

**Discussion by boheme61** [home-assistant/frontend#16311] [Dashboard]
Some people (like me) have a few "LAB/Test" Dashboards, or maybe a Dashboard which one would like to "Disable" temporary, or for longer periods(for some reasons)
LAB/Test Dashboards tend to commute high usage, with all card/template/graph "tests" , but one surely don't starts to delete much there(always good to have "nearby" ) , other Dashboards might be irrelevant for a period, and others maybe for Temporary Users/ or uses ( well there must be others that have some reasons )

The thing is, everything is loaded, upon start/restart, cards/scripts/templates etc ( and who knows what )

Would be great if one could "Disable" a Dashboard, in i.e /Settings/Dashboards, so this wont affect Usage And/or why not stability / functionality in common ... would also be nice in "Troubleshooting" purposes, ( Disable a dashboard, if one things it causes above mentioned issues )

**Comment by ildar170975** [comment]
Assume there there 2 dashboards, each of then has 10 views.
When some view of some dashboard is open - are resources / cards / js / whatever for other views loaded?
